packs/feats/celestial-magic.json

```diff
@@ -28,7 +28,48 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    {
+                        "or": [
+                            "item:slug:clear-mind",
+                            "item:slug:sure-footing",
+                            "item:slug:share-life",
+                            "item:slug:revealing-light",
+                            "item:slug:humanoid-form",
+                            "item:slug:everlight"
+                        ]
+                    },
+                    "spellcasting:innate"
```

Comment:
We should do a choice set for the spell. See my example in the data entry channel

Reply:
Choice set added - please check when you get a moment.

Thanks!